bash/README.md

```diff
@@ -0,0 +1,27 @@
+# UNIX and bash
+
+Resources for the UNIX and bash session:
+
+**Instructor:**
+[Paul Wessel](http://www.soest.hawaii.edu/wessel/)
+
+* My Google Slides on [UNIX and bash](https://docs.google.com/presentation/d/16zYg13TOp9u_raN04BFJnENABHi-tKGzBXQWLtg87oA/edit?usp=sharing)
+* Learn more at [Software Carpentry](http://swcarpentry.github.io/shell-novice)
+* Will be using [Visual Studio Code](https://code.visualstudio.com/) with a split screen editor/terminal layout (Select Terminal->New Terminal)
+
+**Tasks**
+
+* Survey the most basic bash command for doing work in a terminal
+* Explore our directory tree from the command line
+* Make directories and delete them from the command line
+* Create a helper script to make a fresh gmt bash script that is executable.
+
+**Common Problems**:
+
+* Your PATH does not contain "." so you must type ./yourscript.sh to run your script.
```

Comment:
Since most people will have this, it's probably best to just type `./` every time to avoid confusion. They can figure out how to avoid 2 extra characters later on.